src/ui/units/workbooks/components/WorkbookPage/WorkbookPage.tsx

```diff
@@ -58,17 +63,23 @@ export const WorkbookPage = () => {
         return queryTab ? (queryTab as TabId) : undefined;
     }, [search]);
```

Comment:
queryTab  already contains information about TAB_ALL, I suggest leaving isMainTab, as it will be needed to separate the components on WorkbookMainTabContent and WorkbookTabContent